csrankings.csv

```diff
@@ -20716,4 +20716,4 @@ Peter Savadjiev,McGill University,http://www.cim.mcgill.ca/~petersv/,rve3f1MAAAA
 Tal Arbel,McGill University,http://www.cim.mcgill.ca/~arbel/,0AN34NMAAAAJ
 Michael G. Rabbat,McGill University,https://research.fb.com/people/rabbat-mike/,cMPKe9UAAAAJ
 AJung Moon,McGill University,https://www.ajungmoon.com/home,NNnTnK8AAAAJ
-Aditya Mahajan,McGill University,http://www.ece.mcgill.ca/~amahaj1/,Qr6-idkAAAAJ
+Aditya Mahajan,McGill University,http://www.ece.mcgill.ca/~amahaj1/,Qr6-idkAAAAJ
```

Comment:
ECE, not CS. Not eligible for inclusion.

Reply:
Practically all of these folks (on a spot check) do not appear on the CS directory: https://www.cs.mcgill.ca/people/faculty/

Reply:
They supervise students who are admitted to the CS department at McGill. I believe it is a useful information for potential students who want to apply to McGill CS.

Reply:
There is no indication that these folks are cross-appointed or can _solely_ supervise PhD students for a CS degree. I need something concrete.

Reply:
> As mentioned above, tenure-track faculty who can advise PhD students in CS can be included regardless of their home department. The primary audience of CSRankings is prospective graduate students who are seeking a postgraduate degree in Computer Science.

This is from the FAQ page. http://www.cim.mcgill.ca/~pvg/ you can find the list of students that a Prof from ECE (in this case Tal Arbel) is supervising students admitted to CS.